onboarding/onboarding.md

```diff
@@ -0,0 +1,221 @@
+- Het elkaar verantwoordelijk houden als professionals.
+
+Developers die aan Ampersand werken of hebben gewerkt:
+
+| Naam | Project(en) | Start | Eind |
+| ---- | ------- | ----- | ---- |
+| Andre de Nijs | RAP, Front-end | 01/09/2022 | - |
+| Bas Plijnaer | ?? | ?? | ?? |
+| *Dirk Suelmann* | Front-end | 07/11/2022 | 16/12/2022 |
+| Freek Rodenburg | RAP deployment | 07/11/2022 | - |
+| *Jesse Zwitserlood* | ?? | ?? | ?? |
+| Kemal Yildiz | Testen RAP | 07/11/2022 | 30/12/2022 |
+| Li-Wei Yeh | Front-end | 01/09/2022 | - |
+| *Mark van der Hart* | Front-end | 01/09/2022 | 30/11/2022 |
+| *Naomi Prins* | Front-end | 07/11/2022 | 16/12/2022 |
+| Patrick Ramge | Front-end, RAP | 01/09/2022 | 30/12/2022 |
+| Sheriff Balunywa | Front-end | 16/12/2022 | - |
+| Sharvin Ramdat | Testen RAP, Testen front-end | ?? | - |
+
+## Product owner(s)
+
+
+Product owners Ampersand project:
+- [Stef Joosten](mailto:stef.joosten@ordina.nl)
+- [Han Joosten](mailto:han.joosten@ordina.nl)
+- [Michiel Stornebrink](mailto:michiel.stornebrink@tno.nl)
+
+## Scrum master
+De Scrum Master is verantwoordelijk voor het opzetten van Scrum, zoals staat beschreven in de Scrum Gids. Scrum Masters doen dit door iedereen te helpen om Scrum theorie en praktijk te begrijpen, zowel binnen het Scrum Team als binnen de organisatie.
+
+De Scrum Master is verantwoordelijk voor de effectiviteit van het Scrum Team. Scrum Masters doen dit door het Scrum Team in staat te stellen zijn werkwijzen te verbeteren binnen het Scrum raamwerk.
+
+In het Ampersand project is geen vast scrum master, deze wisselt daarom per sprint. Van de developer die de rol scrum master wordt toebedeeld, wordt verwacht dat deze hier tijd voor inpland en dit als een leerdoel ziet.
+
+| Sprint | Start | Eind | Scrum master |
+| ---- | ------- | ----- | ---- |
+| 8 | 21/11/2022 | 02/12/2022 | Li-Wei Yeh |
+| 9 | 05/12/2022 | 16/12/2022 | Freek Rodenburg |
+| 10a | 19/12/2022 | 30/12/2022 | Patrick Ramge |
+| 10b | 02/01/2023 | 15/01/2023 | Li-Wei Yeh |
+
+## Business Analyst
+De Business Analyst heeft als taak systemen en processen te verbeteren. Zij doet dit door de brug te zijn tussen de product owners en developers. Taken af te stemmen en het backlog te beheren.
+
+Business analisten die aan Ampersand werken of hebben gewerkt:
+
+| Naam | Start | Eind | Uren /week |
+| ---- | ----- | ---- | ---------- |
+| Manon de Willigen | 12/12/2022 | - | 4 |
+
+## Belanghebbenden / stakeholders
+Elk project heeft belanghebbenden, bijvoorbeeld de opdrachtgever en/of gebruikers. 
+
+Belanghebbenden van het Ampersand project:
+- Open Universiteit: vertegenwoordigd door [Stef Joosten](mailto:stef.joosten@ordina.nl)
+- TNO: vertegenwoordig door [Michiel Stornebrink](mailto:michiel.stornebrink@tno.nl)
+- Ordina: vertegenwoordigd door [Stef Joosten](mailto:stef.joosten@ordina.nl), [Han Joosten](mailto:han.joosten@ordina.nl) en [Wilco Morren](mailto:wilco.morren@ordina.nl)
```

Comment:
Vanuit Ordina geen stakeholders. Han & Stef vervullen hun rol als stakeholder als externen, ondanks het collegas zijn. Wilco is alleen verantwoordelijk voor het A-Team